src/redux/sagas/multiSig/createRootMultiSig.ts

```diff
@@ -33,7 +34,7 @@ function* createRootMultiSigSaga({
     annotationMessage,
     customActionTitle,
     domainId = Id.RootDomain,
-    requiredRole = ColonyRole.Root,
+    requiredRole = UserRole.Owner,
```

Comment:
What do you think about having a map between `operationName` and required roles? It would hide that implementation detail from the UI

Reply:
nice idea, will do just that!